src/test/incremental/hashes/call_expressions.rs

```diff
@@ -151,8 +151,8 @@ pub fn change_to_ufcs() {
 #[cfg(not(cfail1))]
 #[rustc_clean(cfg="cfail2", except="HirBody,mir_built,optimized_mir,typeck_tables_of")]
 #[rustc_clean(cfg="cfail3")]
-// One might think this would be expanded in the HirBody/Mir, but it actually
-// results in slightly different Hir/Mir.
+// One might think this would be expanded in the HirBody/Body, but it actually
```

Comment:
Maybe talk about `mir::Body` to make it clear what is being talked about

Reply:
`HirBody` might be a weird depnode syntax, I think it should say "expanded in {hir,mir}::Body" and "slightly different HIR/MIR"

Reply:
k!
I'm gonna put this in the same commit where I change the rest of the docs.